packages/client/README.md

```diff
@@ -230,6 +237,24 @@ const domainScopedClient = OpenFeature.getClient("my-domain");
 Domains can be defined on a provider during registration.
 For more details, please refer to the [providers](#providers) section.
 
+#### Manage evaluation context for domains
+
+By default, domain-scoped clients use the global context.
+This can be overridden by explicitly setting context when registering the provider or by references the domain when updating context.
+
+```ts
+OpenFeature.setProvider("my-domain", new NewCachedProvider(), { isCache: true });
```

Comment:
Could we also set the targeting key here? I think this makes the example more obvious.
`isCache: true` can be confused for options or so.

Reply:
Ya, I agree. I think something more personal like "`email`" might be clearer.